make/photon/registry/Dockerfile.binary

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.11
+FROM golang:1.12.12
```

Comment:
The version of golang is just referring to the upstream, do we really need to upgrade it?

Same as Notary and Clair, please double confirm.

Reply:
I think yes, we need to patch the artifacts we build using golang.  Any disagreement?